portfolio/src/main/java/com/google/sps/authentication/servlets/LoginServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.authentication.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+    private UserService userService;
+
+    @Override
+    public void init() {
+        this.userService = UserServiceFactory.getUserService();
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Does the tutorial suggest using get or something else? How would the password be transmitted?

Reply:
hmmm the tutorial does not mention... actually when direct to the log in page, only email address is needed, no place for password

Reply:
The example use GET

Reply:
Fixed with adding comments as discussed offline